crates/ruma-identifiers/src/matrix_uri.rs

```diff
@@ -47,7 +47,7 @@ pub enum MatrixId {
     User(Box<UserId>),
 
     /// An event ID.
-    Event { room_id: Box<RoomOrAliasId>, event_id: Box<EventId> },
+    Event((Box<RoomOrAliasId>, Box<EventId>)),
```

Comment:
Sorry, I meant a variant with two unnamed fields, not a variant with a single unnamed tuple field.